riscv/csrs.cc

```diff
@@ -1181,23 +1181,24 @@ bool composite_csr_t::unlogged_write(const reg_t val) noexcept {
 }
 
 
-sentropy_csr_t::sentropy_csr_t(processor_t* const proc, const reg_t addr):
+seed_csr_t::seed_csr_t(processor_t* const proc, const reg_t addr):
   csr_t(proc, addr) {
 }
 
-void sentropy_csr_t::verify_permissions(insn_t insn, bool write) const {
+void seed_csr_t::verify_permissions(insn_t insn, bool write) const {
   /* Read-only access disallowed due to wipe-on-read side effect */
+  /* XXX mseccfg.sseed and mseccfg.useed should be verified. */
```

Comment:
Just to confirm: the reason you think it's OK to commit this `XXX` is that `mseccfg` hasn't been implemented yet, and so this is legitimately future work.  I'm OK with this.

Reply:
Yes once mseccfg is there, we will add those access control features in a consistent way.

Reply:
> Just to confirm: the reason you think it's OK to commit this `XXX` is that `mseccfg` hasn't been implemented yet, and so this is legitimately future work. I'm OK with this.

Yes once mseccfg is there, we will add those access control features in a consistent way.